src/apps/chifra/cmd/receipts.go

```diff
@@ -57,3 +60,46 @@ func init() {
 
 	chifraCmd.AddCommand(receiptsCmd)
 }
+
+// `runWithFileSupport` runs a command in the usual way unless `--file` is specified.
+// If it is specified, this function will parse the file and then run the command
+// in series of independent calls (just like calling `chifra` N times on the command line,
+// but without wasting time and resources for the startup)
+func runWithFileSupport(cmd *cobra.Command, args []string) error {
+	// try to open the file
+	filePath, err := cmd.Flags().GetString("file")
+	if err != nil {
+		return err
+	}
+	if filePath == "" {
+		// `--file` has not been provided, run the command as usual
+		return receiptsPkg.RunReceipts(cmd, args)
+	}
+
+	// parse commands file
+	commandsFile, err := file.ParseCommandsFile(cmd, filePath)
+	if err != nil {
+		return err
+	}
+
+	for _, line := range commandsFile.Lines {
+		receiptsPkg.ResetOptions()
```

Comment:
This could be a function pointer to make `runWtihFileSupport` available to all commands.